h2o-r/h2o-package/R/models.R

```diff
@@ -3439,6 +3785,20 @@ h2o.cross_validation_holdout_predictions <- function(object) {
 #'
 #' @param object An \linkS4class{H2OModel} object.
 #' @return Returns a list of H2OFrame objects
+#' @examples 
```

Comment:
This cross_validation_predictions example is missing the opening and closing { }